### PR TITLE
Use defaultTraceConfiguration instead of traceConfiguration

### PR DESCRIPTION
**Pull request by TADraeseke**
Hey @Rachael-E @shubham7109 -- can you review when you have a chance? Thanks!

### CODE REVIEW

File: kotlin/trace-utility-network/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Tap on one or more features while 'Add starting locations' or 'Add barriers' is
 9.  If an edge, set its `FractionAlongLine` property using `GeometryEngine.fractionAlongEdge`.
 10. Add this `UtilityElement` to a collection of starting locations or barriers.
 11. Create `TraceParameters` with the selected trace type along with the collected starting locations and barriers (if applicable). 
-12. Set the `TraceParameters.traceConfiguration` with the utility tier's `TraceConfiguration` property.
+12. Set the `TraceParameters.defaultTraceConfiguration` with the utility tier's `TraceConfiguration` property.
```
Comment (Rachael-E) [inline kotlin/trace-utility-network/README.md:29]: this doesn't need updated: the traceconfiguration method on TraceParameters has not been deprecated :)

**Review by Rachael-E — APPROVED** [review]
thanks @TADraeseke , just one small change to make since the trace configuration method on the trace parameters class isn't being deprecated it would seem! will approve so as not to hold you up once that change to the readme is reverted.

File: kotlin/trace-utility-network/README.md (hunk 1 above)
Comment (TADraeseke) [reply kotlin/trace-utility-network/README.md:29]: Ahh good spot, thanks!

**Review by shubham7109 — APPROVED** [review]
Looks good!

**Pull request merged**